posthog/temporal/workflows/squash_person_overrides.py

```diff
@@ -185,24 +185,25 @@ class QueryInputs:
     password: str = ""
     cluster_name: str = ""
     dictionary_name: str = "person_overrides_join_dict"
-    _latest_created_at: str | datetime | None = None
+    dry_run: bool = False
```

Comment:
Let's default to true here instead, it's safer to accidentally run in dry_run mode rather than the other way around (& test without providing the val)